src/redux/api.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+const baseUrl = 'https://comic-dance-club-api.herokuapp.com';
+
+const fetchDataClasses = async () => {
+  try {
+    const response = await axios.get(`${baseUrl}/classes/`);
```

Comment:
- [x] This end point does not exist
You can use:
* /api/items
* /api/reservations